contributors/devel/sig-architecture/api-conventions.md

```diff
@@ -1145,11 +1145,11 @@ reduce data volume, load on the system, and noise exposed to users.
 
 ## Naming conventions
 
-* Go field names must be CamelCase. JSON field names must be camelCase. Other
+* Go field names must be PascalCase. JSON field names must be camelCase. Other
```

Comment:
Thinking about this, resource names (eg CronJob) are PascalCase, but the fields inside those resources are camelCase (eg `apiVersion`).

Reply:
So @sftim, would you think to avoid any discrepancies we'd require for it to be explicitly mentioned/added somewhere? Or highlighting it so that it's more visible?

Reply:
I think it's worth clarifying so that people can tell which convention applies where.

Reply:
I think we can merge the change so it corrects the error, and then look at a second PR that adds more clarification.

Reply:
Now that this has merged, I invite any interested parties to collaborate with me on a second PR that answers the issues I've raised in [issue 20862](https://github.com/kubernetes/website/issues/20862). If you understand Kubernetes capitalization customs, let's figure out how to codify them.